draft-pauly-add-requirements.md

```diff
@@ -186,68 +186,82 @@ Using a content-provider's encrypted resolver can also provide several privacy a
 and the content provider, assuming that the content provider would already see the names upon a secure connection
 later being made based on the DNS answers (e.g., in the TLS SNI extension)
 
-# Distilled discovery mechanisms {#mechanisms}
+# Discovery mechanisms {#mechanisms}
 
-The various use cases described in {{use-cases}} do not all necessarily require separate mechanisms.
+The use cases described in {{use-cases}} do not all necessarily require separate mechanisms.
 
-Generally, the use cases can be summarized in two catgories:
+Generally, the use cases can be summarized in two categories:
 
-1. Discover encrypted resolvers equivalent to (or associated with) unencrypted resolvers. (Network-provisioned,
-client-selected, VPN)
-2. Discover encrypted resolvers applicability to a limited set of domains. (Private names, local content, CDN content)
+1. Resolver upgrade: Discover encrypted resolvers equivalent to (or associated with) unencrypted resolvers.
+Examples include network-provisioned, client-selected, and VPN-configured resolvers.
+2. Domain-specific resolvers: Discover encrypted resolvers applicable to a limited set of domains.
+Examples include resolvers for enterprise or private names, local content, and CDN content.
 
-For the first group, mechanisms to discover equivalent resolvers can either add new parameters to existing provisioning
+Resolver upgrade mechanisms can either add new parameters to existing provisioning
 mechanisms (adding necessary information to use DoT or DoH to options in DHCP, RAs, or IKEv2) or else provide a way
 to communicate with a provisioned unencrypted DNS resolver and discover the equivalent or associated encrypted
 DNS resolver.
 
-For the second group, mechanisms additionally need to provide some information about the applicability and capabilities
-of encrypted resolvers. This information could either be provisioned,
+Domain-specific resolver discovery mechanisms additionally need to provide some information about the
+applicability and capabilities of encrypted resolvers. This information could either be provisioned,
 or be discovered based on clients actively trying to access content.
 
 # Privacy and security requirements {#priv-sec}
 
-Since one of the main reasons for enabling encrypted DNS on clients is to improve the privacy and security properties
-of DNS queries, discovery mechanisms must be designed carefully to not weaken a client's security or privacy.
+A primary goal of encrypted DNS is improving the privacy and security of DNS queries and answers in the presence
```

Comment:
```suggestion
Encrypted DNS aims to improve the privacy and security of DNS queries and answers in the presence
```

Reply:
Passive voice!

Reply:
Are you saying "X aims <infinitive>" is more passive than "A goal of X is ___"? That seems backwards to me, but you can always change it to just "Encrypted DNS improves" if you want to be maximally active. :)

Reply:
"To improve" :) "X improves" is better!